src/org/zaproxy/zap/extension/pscanrulesBeta/ServletParameterPollutionScanner.java

```diff
@@ -75,20 +73,10 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 			// check for 'target' param
 			
 			for (Element formElement : formElements) {
-				boolean actionFound = false;
-				Attributes atts = formElement.getAttributes();
-				Iterator<Attribute> iter = atts.iterator();
-				
-				while (iter.hasNext()) {
-					Attribute att = iter.next();
-					if (att.getName().equalsIgnoreCase("action") && att.getValue().length() > 0) {
-						// action tag present (and with a value), so should be ok
-						actionFound = true;
-					}
-				}
-				
-				if (!actionFound) {
-				    Alert alert = new Alert(getPluginId(), Alert.RISK_MEDIUM, Alert.CONFIDENCE_LOW, 
+				boolean actionMissingOrEmpty = StringUtils.isEmpty(formElement.getAttributeValue("action"));
+
+				if (actionMissingOrEmpty) {
```

Comment:
``` isEmpty ``` deals with that and I have a test https://github.com/zaproxy/zap-extensions/pull/531/files#diff-64aeafc22cf8462053c5cd606308af96R55 which covers this situation.

I had to make this change because the previous wasn't checking ``` att.getValue() ``` before using it and causing the test failure.